<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="MainListWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>155</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>220</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="MainVerticalLayout"/>
   </widget>
   <widget class="QPushButton" name="btnClean">
    <property name="geometry">
     <rect>
      <x>503</x>
      <y>330</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear scene</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>491</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Sphere</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cone</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hyperboloid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cylinder</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Torus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mobius</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Triangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plane</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Disk</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
